documentation/src/test/java/org/hibernate/search/documentation/mapper/orm/gettingstarted/withhsearch/customanalysis/MyElasticsearchAnalysisConfigurer.java

```diff
@@ -4,9 +4,9 @@
  * License: GNU Lesser General Public License (LGPL), version 2.1 or later
  * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
  */
-// tag::include[]
 package org.hibernate.search.documentation.mapper.orm.gettingstarted.withhsearch.customanalysis;
 
+// tag::include[]
```

Comment:
I think the package was included here because we refer to the FQCN in configuration properties. So it makes sense to show the package name is the same.

Same for other examples.